### DISCUSSION TITLE
Seems that some unusual accounts followed me and forked my repository

### DISCUSSION THREAD

**Discussion by fzyzcjy** [community/community#53643] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When viewing my homepage, I notice some unusual things today:

<details>

![image](https://user-images.githubusercontent.com/5236035/234141676-823ad23c-e7a4-4e03-afa8-48108ba10c49.png)

</details>

The pattern seems a bit unusual unlike how I normally gets stars/follows from real users. So I checked them:

<details>

![image](https://user-images.githubusercontent.com/5236035/234142252-5fe3edcd-3776-4bc5-9544-50d181f82f18.png)
![image](https://user-images.githubusercontent.com/5236035/234142319-171ea492-92b5-4de4-b580-bdc92e5b26ae.png)
![image](https://user-images.githubusercontent.com/5236035/234142417-27f2777d-baa1-4284-b143-bcf057cb5ade.png)
![image](https://user-images.githubusercontent.com/5236035/234142556-da7b8de0-57c5-408d-95ab-da072c4f1511.png)

whose twitter link (https://twitter.com/alegaoanch) is banned.

![image](https://user-images.githubusercontent.com/5236035/234142820-3591e412-dea9-4346-a596-1e14e761c4f6.png)

</details>

The reason I feel weird is that:

1. These accounts are registered only a few hours ago
2. One account is linked to a Twitter account that has been banned by Twitter (the https://twitter.com/alegaoanch)
3. Except for when I post my repos on Reddit/HN etc, I seldom see that I get this many new follows/forks in such a short duration.

I want to believe that they are just new users of GitHub, and it is just randomness that brings this many registered-a-few-hours accounts to follow and fork me, and if I wrongly suspect them while they are real user, I do apologize for that!

However, the reason why I post this thread is that, I have heard that some people are creating fake accounts on GitHub and pay money to get fake stars, or use any approach to get the stars that should not have belong to them. I **HATE** that very much, which makes the stars on GitHub meaningless, if such behavior become a standard. I do value the stars/watches/forks I get, because I think that can be a measure of how many people are using/loving my library and I am proud that my tooling can help many people.

Therefore, when I seeing the unusual accounts above, I suspect it is some account that will be used to such pay-to-get-fake-stars behavior above. Thus, (1) I do hope such pay-to-get-stars accounts (if they are really fake and pay-to-get-stars account, which offenses GitHub policy) can be banned (2) I do hope no such fake accounts give stars/forks to my own repository,.

Thanks for GitHub that is quite helpful as my everyday tool. If I should post on GitHub support or somewhere else, please tell me :)

(p.s. I am using the open-source hosted-on-github Refined GitHub plugin so the screenshots may be a little bit different from what you view it, but the things are mainly the same)

---

EDIT: I see a bit more after posing this discussion (current time is 11:50 am Tuesday, 25 April 2023 @ UTC timezone), which is shown as follows. There are also some normal accounts so I just optionally screenshot the seemingly unusual accounts.

<details>

![image](https://user-images.githubusercontent.com/5236035/234269618-886dd5a0-de0e-4d50-b66b-0cb44512a6e3.png)
![image](https://user-images.githubusercontent.com/5236035/234270207-045af9ba-38bd-4351-9891-a590a6fcad4c.png)

![image](https://user-images.githubusercontent.com/5236035/234269823-d257eedc-c48d-42ee-8223-d498e3d538d8.png)
![image](https://user-images.githubusercontent.com/5236035/234270131-f533c413-123c-47d3-b49f-f6a8e42b9ea6.png)

</details>

---

EDIT2: One more (mainly the second account which is also registered within a few hours)

<details>

![image](https://user-images.githubusercontent.com/5236035/234308293-36625aa2-a4d4-4cc0-b864-d68084bc0481.png)
![image](https://user-images.githubusercontent.com/5236035/234308338-6324e930-a136-4183-aae9-e6b9338f2883.png)

</details>

---

EDIT 3: Realized I seem to have missed out one account, who forked 3 repo and followed me and registered yesterday and has unusual twitter content - he says `s of the brands target market78SMS Marketing:  M\nFri Feb  3 17:56:26 2023`.

<details>

![image](https://user-images.githubusercontent.com/5236035/234314485-40caed37-fa9f-4be3-b1fa-d4c4ba15f3c4.png)
![image](https://user-images.githubusercontent.com/5236035/234314603-5fac6dd5-b5ed-4fdf-98cf-7938534a8d4a.png)

![image](https://user-images.githubusercontent.com/5236035/234315900-f7cb8bb8-ab43-46b0-860c-6bfc03041ebb.png)

</details>

---

EDIT 4: Spent a few minutes to take screenshot of the "following users" page. Seems these accounts follow many same accounts, which I am not very sure whether it is normal or unusual, so just paste it here for GitHub staff analysis.

<details>

![image](https://user-images.githubusercontent.com/5236035/234313603-8ff27f66-f6e7-4166-b667-4483ccc52b9c.png)
![image](https://user-images.githubusercontent.com/5236035/234313626-e17800f6-0ef6-474d-b203-b548fecbcc22.png)

![image](https://user-images.githubusercontent.com/5236035/234313927-5652c9ff-cde4-44f3-acb0-5ce1f65d82ee.png)

![image](https://user-images.githubusercontent.com/5236035/234313963-e890d15f-d24f-4a71-8316-780d252a6953.png)

![image](https://user-images.githubusercontent.com/5236035/234314730-c1c4bc6c-f058-4d8c-90da-d16c9e95ca6f.png)
![image](https://user-images.githubusercontent.com/5236035/234314762-7bb96d5c-2f73-4578-ba6c-b310e432fe97.png)
![image](https://user-images.githubusercontent.com/5236035/234314795-8932ca51-b2ca-4e0e-b54e-5a9419de5dd7.png)
![image](https://user-images.githubusercontent.com/5236035/234314828-1612f573-3464-4adb-b0dc-051247e02879.png)

![image](https://user-images.githubusercontent.com/5236035/234314858-bf7aa413-cfc8-4e69-b535-3b9191c9b8a7.png)

![image](https://user-images.githubusercontent.com/5236035/234314910-28505ca8-911e-4afd-a7c7-67f1773f3eec.png)

![image](https://user-images.githubusercontent.com/5236035/234315129-ca4afb23-5006-4c1d-8f7f-1432c1324322.png)

![image](https://user-images.githubusercontent.com/5236035/234315168-36009a4e-cf05-49af-bcba-0b014d34b2f5.png)

![image](https://user-images.githubusercontent.com/5236035/234315266-db091d5f-ce16-44ef-9e12-0ba56de2d14e.png)

</details>

---

EDIT 5: Seems one more account that is registered a few hours ago

<details>

![image](https://user-images.githubusercontent.com/5236035/234440622-8e043cf1-49c3-4a6d-83e7-ccf1668d85c7.png)
![image](https://user-images.githubusercontent.com/5236035/234440560-4fd2f86a-b29a-448c-82ba-8e76d79e4c07.png)
![image](https://user-images.githubusercontent.com/5236035/234440652-a16d9286-8427-4176-a808-7974f25b64d6.png)

</details>

**Comment by Yooo-Its-King** [comment]
I would definitely contact support from these companies. Though I don't see the problem with people starring your repositories. What you've described is very suspicious. Again, I don't see how things like new follows are a problem, but, I believe you can turn off the option for others to fork your main branch in your account settings.

**Reply by fzyzcjy** [reply]
Thanks, IMHO this "discussion forum" is the official GitHub one and GitHub people have replied here frequently.

**Reply by Yooo-Its-King** [reply]
Yeah, people are pretty active in this discussion forum. Users are fortunate.

**Reply by ghost** [reply]
**lol**

**Comment by the1Riddle** [comment]
I'm surprised to read that some fake accounts pay for stars, now I get why you are so suspicious.

**Comment by sametaor** [comment]
I was also getting suspicious of this, will defo review the follower activity now.

**Comment by ghost** [comment]
Oh I thought I was the only one. Some seem like bots or AI accounts. How do I fix this? It is concerning.

**Comment by ullaskunder3** [comment]
there is something wrong
<img width="1107" height="973" alt="image" src="https://github.com/user-attachments/assets/f07b9922-527a-420f-93ae-b8f8908ddaa4" />

**Reply by ndizeyedavid** [reply]
I am getting spam accounts like these following me too. I try blocking them every time i see one, but they keep on coming every 2 weeks

**Reply by ullaskunder3** [reply]
its a mess
